assignment-2/src/presenter/loginPresenter.js

```diff
@@ -0,0 +1,33 @@
+import user from "../model/Users";
+
+class QuestionsListPresenter {
+    onLogin(){
+        let newUser = user.state.newUser;
+        let users = user.state.users;
+
+        for (let i = 0; i < users.length; i++) {
```

Comment:
This belongs to the model and not inside the presenter (as it is more of a business / functional operation).